core/chains/evm/client/errors.go

```diff
@@ -108,7 +108,29 @@ var avalanche = ClientErrors{
 	NonceTooLow: regexp.MustCompile(`(: |^)nonce too low: address 0x[0-9a-fA-F]{40} current nonce \([\d]+\) > tx nonce \([\d]+\)$`),
 }
 
-var clients = []ClientErrors{parity, geth, arbitrum, optimism, substrate, avalanche}
+// Nethermind
+// All errors: https://github.com/NethermindEth/nethermind/blob/master/src/Nethermind/Nethermind.TxPool/AcceptTxResult.cs
+// All filters: https://github.com/NethermindEth/nethermind/tree/9b68ec048c65f4b44fb863164c0dec3f7780d820/src/Nethermind/Nethermind.TxPool/Filters
+var nethermindFatal = regexp.MustCompile(`(: |^)(SenderIsContract|Invalid|Int256Overflow|FailedToResolveSender|NonceGap|GasLimitExceeded)`)
+var nethermind = ClientErrors{
+	// OldNonce: The EOA (externally owned account) that signed this transaction (sender) has already signed and executed a transaction with the same nonce.
+	NonceTooLow: regexp.MustCompile(`(: |^)OldNonce`),
```

Comment:
Can we make it `(: |^)OldNonce$` ? And so for all these errors?

Reply:
Sure thing!